src/Domain.jl

```diff
@@ -114,6 +114,7 @@ struct ConstantVDomain{N<:AbstractPhase,S<:Integer,W<:Real,W2<:Real,I<:Integer,Q
     sensitivity::Bool
     jacuptodate::MArray{Tuple{1},Bool,1,1}
     t::MArray{Tuple{1},W2,1,1}
+    p::Array{W,1}
```

Comment:
Now I see that this p is added back. I think this can probably squashed with the previous commit

Reply:
done